src/directives/repeat-click.js

```diff
@@ -1,12 +1,30 @@
 import { once, on } from 'element-ui/src/utils/dom';
 
+function getRepeatDuration(bindingArg, vnode, defaultDuration) {
+  let duration = defaultDuration;
+  if (bindingArg) {
+    let arg = bindingArg;
+    const dynamicArgMatch = typeof arg === 'string' ? /^\[(.*)\]$/.exec(arg) : null;
```

Comment:
I found that the dynamic argument approach in the Vue.js documentation didn't work automatically, but could be made do work using the `vnode` argument (already used for the function/expression below).
Also, here's a SO post explaining the approach: https://stackoverflow.com/questions/45223018/how-to-pass-dynamic-argument-in-vuejs-custom-directive/48631316#48631316
And, here's a post where Evan You announces it in Vue 2.6: https://medium.com/the-vue-point/vue-2-6-released-66aa6c8e785e#ca69